rules/rules.js

```diff
@@ -108,6 +108,56 @@ const removeRule = function (uid) {
     }
 };
 
+
+/**
+ * Runs the rule with the given UID. Throws errors when the rule doesn't exist
+ * or is unable to run (e.g. it's disabled).
+ * 
+ * @memberOf rules
+ * @param {String} uid the UID of the rule to run
+ * @param {Map<Object>} [args={}] args optional dict of data to pass to the called rule
+ * @param {Boolean} [cond=true] when true, the called rule will only run if it's conditions are met
```

Comment:
should we add a throws line as well?  for example
```javascript
@throws Will throw an error if the rule is not found or is disabled
```

and also in the two functions below ? 

https://jsdoc.app/tags-throws.html

Reply:
I started to but it's not on any of the other methods that throw Errors in this file so I stuck with that as the convention. Should I add @throws to them all or just mine?

Reply:
Thanks!